sdk/ai/azure-ai-client/samples/agents/sample_agents_code_interpreter_attachment.py

```diff
@@ -61,7 +61,7 @@
     # notices that CodeInterpreterToolDefinition as tool must be added or the assistant unable to view the file
     agent = ai_client.agents.create_agent(
         model="gpt-4-1106-preview", name="my-assistant", instructions="You are helpful assistant",
-        tools=[code_interpreter]
+        tools=list(code_interpreter.definitions)
```

Comment:
Why are you changing from List to Sequence?   If you use List, you can then use tools = code_interpreter.definitions.    However, I have been trying to remove setting tools at the create_agent since you probably should pass it to the create_message instead.

Reply:
Oh..   I see the suggestion from changing from LIst to Sequence.

Reply:
We can adjust that still. I agree Sequence may not be ideal, but Pylance error proposed it.